pyproject.toml

```diff
@@ -12,6 +12,7 @@ dependencies = [
     "pycountry~=20.7.3",
     "Unidecode~=1.2.0",
     "barcodenumber~=0.5.0",
+    "rapidfuzz~=2.15.0",
```

Comment:
_Just commenting on dependency_

This has compiled components but wheels seem to be available for most py versions on linux, also seems to be actively maintained so it should be fine.